### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by kasugaijin**
# 🔗 Issue
Some more small fixes

# ✍️ Description

# 📷 Screenshots/Demos

### CODE REVIEW

File: app/components/dashboard_page_component.html.erb [hunk 1]
```diff
@@ -25,7 +25,7 @@
   <% if nav_tabs? %>
     <!-- Nav tabs -->
     <div class="row">
-      <div class="col-12 border-bottom">
```
Comment (kasugaijin) [inline app/components/dashboard_page_component.html.erb:28]: There are two bottom borders on the staff pets show page.

File: app/javascript/controllers/toggle_completed_controller.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { Controller } from "@hotwired/stimulus";
 export default class ToggleCompletedController extends Controller {
   connect() {
     this.completedTasks = document.querySelectorAll('.bg-success-subtle');
+    this.hideCompletedTasks();
```
Comment (kasugaijin) [inline app/javascript/controllers/toggle_completed_controller.js:6]: I prefer we hide completed pet tasks by default

File: app/views/organizations/staff/tasks/index.html.erb [hunk 3]
```diff
@@ -22,9 +22,8 @@
                     class="form-check-input" 
                     type="checkbox" 
                     role="switch" 
-                    id="flexSwitchCheckChecked" 
-                    checked 
-                    data-controller="toggle-completed" 
+                    id="flexSwitchCheckChecked"
+                    data-controller="toggle-completed"
```
Comment (kasugaijin) [inline app/views/organizations/staff/tasks/index.html.erb:26]: Remove the checked true from the show completed tasks toggle, as we are hiding tasks by default

File: config/locales/en.yml [hunk 4]
```diff
@@ -464,7 +464,7 @@ en:
           help_due_in_days: "The task will be due this number of days after the pet is created."
         index:
           name: "Name"
-          name_placeholder: "Enter Pet Name"
+          name_placeholder: "Enter task name"
```
Comment (kasugaijin) [inline config/locales/en.yml:467]: Default pet tasks filter fix - the placeholder referred to pet

File: app/components/avatar_component.rb [hunk 5]
```diff
@@ -9,7 +9,7 @@ class AvatarComponent < ApplicationComponent
 
   def avatar
     if image_url
-      image_tag(url(image_url), alt: alt, class: image_classes)
```
Comment (kasugaijin) [inline app/components/avatar_component.rb:12]: This is causing an error in production 
```
NoMethodError (undefined method `url' for an instance of AvatarComponent)
[2ea121ff-ad45-4bc7-b5f3-5ceb98d47fdf]     18:                 <div class="d-flex align-items-end justify-content-between">
[2ea121ff-ad45-4bc7-b5f3-5ceb98d47fdf]     19:                   <div class="d-flex align-items-center">
[2ea121ff-ad45-4bc7-b5f3-5ceb98d47fdf]     20:                     <div class="me-2 position-relative d-flex justify-content-end align-items-end mt-n5">
[2ea121ff-ad45-4bc7-b5f3-5ceb98d47fdf]     21:                       <%= render AvatarComponent.new(current_user, size: :xl) %>
[2ea121ff-ad45-4bc7-b5f3-5ceb98d47fdf]     22:                     </div>
[2ea121ff-ad45-4bc7-b5f3-5ceb98d47fdf]     23:                     <div class="lh-1">
[2ea121ff-ad45-4bc7-b5f3-5ceb98d47fdf]     24:                       <h2 class="mb-0"><%= "#{current_user.full_name}" %></h2>
```

Getting rid of `url` method works fine locally, and I think will fix this issue in production.

**Comment by kasugaijin** [comment]
@jmilljr24 I will leave this PR to this extent for now and pick again tomorrow. It's a number of random small changes from all over the place. Let me know if you have questions. If you get a chance to review, then great. It would be nice to merge these changes then carry on with a new PR for other small stuff.

File: app/components/dashboard_page_component.html.erb (hunk 1 above)
Comment (jmilljr24) [reply app/components/dashboard_page_component.html.erb:28]: Wow! Not sure how to caught that. Nice find.

File: app/javascript/controllers/toggle_completed_controller.js (hunk 2 above)
Comment (jmilljr24) [reply app/javascript/controllers/toggle_completed_controller.js:6]: When I click to view the Tasks tab, the completed tasks are still displayed and the show completed toggle is off. I have to toggle on then back off for the completed tasks to be hidden.

**Review by jmilljr24 — APPROVED** [review]
@kasugaijin good changes. Can you double check the Tasks changes are working as intended?

File: app/javascript/controllers/toggle_completed_controller.js (hunk 2 above)
Comment (kasugaijin) [reply app/javascript/controllers/toggle_completed_controller.js:6]: Can you double check your assets are loaded properly? Throw a console log in there and see it’s getting hit. I had to manually recompile to make it work (which is another issue to address but I thought would only be in docker)

Comment (jmilljr24) [reply app/javascript/controllers/toggle_completed_controller.js:6]: `rails assets:clobber` seemed to do the trick. Thanks

**Pull request merged**